recipes/pysparse/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pysparse" %}
+{% set version = "1.3_dev" %}
+{% set md5 = "3071ea517b5462888ceba71720f6f4a8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # PyPI doesn't build
+  git_url: git://git.code.sf.net/p/pysparse/git
+  git_tag: 1751931aa5c06429245c3aca3a3ba6d7622ea8b5
+
+build:
+  number: 0
+  skip: true  # [py3k]
+  script:
+    python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
In this project do you need to compile something?

Reply:
yes